src/renderers/shared/fiber/ReactFiberCompleteWork.js

```diff
@@ -95,31 +95,30 @@ exports.completeWork = function(current : ?Fiber, workInProgress : Fiber) : ?Fib
     case FunctionalComponent:
       console.log('/functional component', workInProgress.type.name);
       transferOutput(workInProgress.child, workInProgress);
-      break;
+      return null;
     case ClassComponent:
       console.log('/class component', workInProgress.type.name);
       transferOutput(workInProgress.child, workInProgress);
-      break;
+      return null;
     case HostComponent:
       console.log('/host component', workInProgress.type);
-      break;
+      return null;
     case CoroutineComponent:
       console.log('/coroutine component', workInProgress.input.handler.name);
       return moveCoroutineToHandlerPhase(current, workInProgress);
     case CoroutineHandlerPhase:
       transferOutput(workInProgress.stateNode, workInProgress);
       // Reset the tag to now be a first phase coroutine.
       workInProgress.tag = CoroutineComponent;
-      break;
+      return null;
     case YieldComponent:
       // Does nothing.
-      break;
+      return null;
 
     // Error cases
     case IndeterminateComponent:
       throw new Error('An indeterminate component should have become determinate before completing.');
     default:
```

Comment:
It still doesn't look like Flow supports this yet, but one day it should be able to give an error about nonexhaustiveness if we remove this "default" case and miss a case.